### ISSUE TITLE
tiny bug for example in dataframe-hdf5.rst

### ISSUE THREAD

**Issue by gryBox** [dask/dask#1543]
Small 

When copy pasting code `df = dd.read_hdf(path='my86.h5', key='/*')`  from `dataframe-hdf5.rst` and adjusting for file path a `TypeError` is raised
```
---> 18             df = dd.read_hdf(path=db_flpth, key='/*')


TypeError: read_hdf() got an unexpected keyword argument 'path'
```
when not using the keyword `path` ,`dd.read_hdf(db_flpth, key='/*') works fine.

I can make a small fix to the document if you want to something like this...

```
import dask.dataframe as dd
df = dd.read_hdf('my86.h5', key='/c')
```
```
import dask.dataframe as dd
df = dd.read_hdf('my86.h5', key='/*')
```
```
import dask.dataframe as dd
df = dd.read_hdf('./*.h5', key='/*')
```

Thank you @linwoodc3 for the hdf examples.  They are helpful

**Comment by mrocklin** [comment]
That fix sounds great

**Comment by jcrist** [comment]
Fixed by #1544. Thanks!
